infra/resources/prod/main.tf

```diff
@@ -0,0 +1,50 @@
+terraform {
+  required_providers {
+    azurerm = {
+      source  = "hashicorp/azurerm"
+      version = "~> 4.0"
+    }
+  }
+
+  backend "azurerm" {
+    resource_group_name  = "terraform-state-rg"
+    storage_account_name = "tfproddx"
+    container_name       = "terraform-state"
+    key                  = "dx.resources.prod.tfstate"
+    use_azuread_auth     = true
+  }
+}
+
+provider "azurerm" {
+  features {
+  }
+  storage_use_azuread = true
+}
+
+provider "azurerm" {
+  alias = "prod-io"
+  features {
+  }
+  subscription_id     = local.prod-io
+  storage_use_azuread = true
+}
+
+resource "azurerm_role_definition" "pagopa_opex_contributor" {
```

Comment:
idea: can we enable the custom role for every subscription? (using a wildcard scope)

Reply:
Unfortunately we don't have enough privileges to write roles at management group level